rtos/rtx/TARGET_CORTEX_A/RTX_Conf_CA.c

```diff
@@ -74,7 +74,7 @@
 //   <i> Defines stack size for main thread.
 //   <i> Default: 200
 #ifndef OS_MAINSTKSIZE
- #define OS_MAINSTKSIZE 2048
+ #define OS_MAINSTKSIZE 4096
```

Comment:
should also Default in the comment to be updated, because this is the default value (not yet defined)?